src/App.css

```diff
@@ -1,4 +1,4 @@
-/* src/App.css */
+
```

Comment:
- [ ] The app is not responsive enough and has several overflow problems on the mobile view:


![image](https://user-images.githubusercontent.com/57421823/199834026-97330081-1fa8-4a56-8b2d-6e1285ebe305.png)

![image](https://user-images.githubusercontent.com/57421823/199834355-3aa088a4-d9d7-48c6-a66e-786e90d2eb90.png)


Kindly make your app responsive, so your users can view your app from every screen. 💪 

- [ ] For the desktop view, please try to stick to the given design as much as possible, that way, as shown on the project requirements.

That way, you will gain the ability to write software solutions to business requirements, which will serve you well on your next brand-new job! 💪